tfjs-tfdf/src/tfdf_web_api_client.js

```diff
@@ -22,4 +22,17 @@
 // imported correctly by typescript code.
 const YggdrasilDecisionForests = require('../wasm/inference.js');
 
-exports.tfdfWeb = YggdrasilDecisionForests();
+let tfdfWeb = null;
+
+// Function is used here instead of variable to make loading lazy in case user
+// uses setLocateFile before module is used.
+exports.tfdfWeb = () => {
```

Comment:
Nit: If the user has to call this regardless of whether they set the wasm path, could we add an optional argument that just sets the wasm path for them? Might make it a bit easier to call.
```js
exports.tfdfWeb = (locateFile?) => {
  if (locateFile !== null) {
    exports.setLocateFile(locateFile);
  }
  ...
}
```

This probably also requires changing some types and reordering declarations in this file, but I think it's a more convenient interface.